### PR TITLE
install edxapp code sandboxing

### PR DESCRIPTION
**Pull request by jbau**
@jrbl @sefk (note the pull into edx-west/release)

master version forthcoming.

### CODE REVIEW

File: playbooks/roles/edxapp/tasks/deploy.yml [hunk 1]
```diff
@@ -200,6 +201,51 @@
...
+  tags:
+  - deploy
+  - edxapp-sandbox
+
+- name: put code sandbox into aa-enforce or aa-complain mode, depending on EDXAPP_SANDBOX_ENFORCE
+  command: /usr/sbin/{{ edxapp_aa_command }} /etc/apparmor.d/code.sandbox
+  when: EDXAPP_PYTHON_SANDBOX
+  tags:
+  - deploy
+  - edxapp-sandbox
```
Comment (sefk) [inline playbooks/roles/edxapp/tasks/deploy.yml:248]: I presume you didn't actually write this @jbau, instead this was pulled in from someone else's branch (master?).  Is that right?

Comment (jbau) [inline playbooks/roles/edxapp/tasks/deploy.yml:248]: No, I wrote most of this.

**Comment by jrbl** [comment]
Modulo the fact that I don't understand how supervisor.d or apparmor works, yes, this looks sane and reasonable to me as a first pass. I assume you've tested and that your tests have included logging in to double-check permissions on things like ~SANDBOX_USER and so forth.  So :+1:

File: playbooks/roles/edxapp/tasks/deploy.yml [hunk 2]
```diff
@@ -200,6 +201,51 @@
 
 - name: changing group ownership to www-data for everything in the venv (workaround)
   shell: chgrp -R www-data {{ venv_dir }}
+  tags:
+  - deploy
+
+# need to put this into complain mode, otherwise the pip inside the sandbox venv has no permissions
```
Comment (jrbl) [inline playbooks/roles/edxapp/tasks/deploy.yml]: can you fix this comment?

**Comment by jrbl** [comment]
@jbau just gave me the tour and demo'd this for me. I'm happy, once he fixes the comment. :+1:

**Pull request merged**